aiochclient/client.py

```diff
@@ -101,9 +101,22 @@ def __init__(
         ) as resp:  # type: client.ClientResponse
             return resp.status == 200
 
+    @staticmethod
+    def _prepare_query_params(params={}):
```

Comment:
Please make it None by default with type Optional[dict]

Reply:
Done